cmd/kubetoken/kubetoken.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubetoken
```

Comment:
what about not creating its own package if we just need one file for this command? same approach as other tiny commands

Reply:
Yeah it could be. Do you have a preference?

I think that setting its own package defines clearer boundaries. But as you mention, since it's so small it could just be another file from the `cmd` package. I've got no strong opinions here

Reply:
IMO if it is not strictly necessary we should be consistent with how commands have been organised until now in cmd package

Reply:
Yeah, I think we should aim for consistency. The thing is that ATM there's a mix of files and packages. I think we should have a folder for each command in cmd in the long term. 

If we go for one file for each cmd, we might need to refactor the code to move it to the `pkg` folder.

Is there a current decision around this?